lapack/gonum/dgghrd.go

```diff
@@ -10,57 +10,45 @@ import (
 	"gonum.org/v1/gonum/lapack"
 )
 
-// Dgghrd reduces a pair of real matrices (A,B) to generalized upper
-// Hessenberg form using orthogonal transformations, where A is a
-// general matrix and B is upper triangular.  The form of the
-// generalized eigenvalue problem is
```

Comment:
I've removed references to the generalized eigenvalue problem from the doc comment because it's not essential for what Dgghrd does.